bitnami/kube-prometheus/Chart.yaml

```diff
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/prometheus
   - https://github.com/bitnami/containers/tree/main/bitnami/alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 8.5.0
```

Comment:
As this PR adds new features for the chart, could you please bump a minor version `8.6.0`?

Reply:
Hi @migruiz4, just done